developing/cycle/tracker.md

```diff
@@ -0,0 +1,7 @@
+# [Development Cycle](./README.md) / Pivotal Tracker
+
+## Overview
+We use Pivotal Tracker to track tasks and estimate how much time each task will take.
+
+* [How to organize a Pivotal project](https://docs.google.com/document/d/1abmxQdYol6Zi3kR3U__JNJl3uPdKF9WPYWfu75AkZ50/edit#heading=h.yyb7wuihugn8)
```

Comment:
If this is an internal Truss GDoc, I would suggest putting 🔒 next to it to designate it as such.

Reply:
Good call.